faiss.py

```diff
@@ -0,0 +1,35 @@
+import pickle
+import faiss
+
+class FAISS_index:
+
+  def __init__(self, dim, param, nprobe=5):
+    self.index = faiss.index_factory(dim, param)
+    self.dim = dim
+    self.index.nprobe = nprobe
+  
```

Comment:
Could you please add doc strings to these methods? For me it's not really obvious what they actually do.